src/transformers/integrations/integration_utils.py

```diff
@@ -786,15 +786,10 @@ def setup(self, args, state, model, **kwargs):
             self._wandb.run._label(code="transformers_trainer")
 
             # add number of model parameters to wandb config
-            if any(
-                (
-                    isinstance(model, PreTrainedModel),
-                    isinstance(model, PushToHubMixin),
-                    (is_tf_available() and isinstance(model, TFPreTrainedModel)),
-                    (is_torch_available() and isinstance(model, torch.nn.Module)),
-                )
-            ):
+            try:
                 self._wandb.config["model/num_parameters"] = model.num_parameters()
+            except AttributeError:
+                pass
```

Comment:
Could we add a logger.info here (feel free to change the message)? By default in transformers info level isn't logged, but might be useful if someone goes into debug mode 

```suggestion
                logger.info("Unable to get num_parameters from model")
```

Reply:
Good call, resolved via a6d09f523bf9241480e9d86f41c2a0ad4db880d2